README.md

```diff
@@ -21,13 +21,13 @@ It creates a JCR user if the user does not exist yet.
 It connects the user as a result of the open authentication.
 
 ### HOW TO USE IT?
-Once you have downloaded Jahia OAuth and at least one connector:
-* Go to your `site > site settings > Jahia OAuth`
+Once you have downloaded Jahia Authentication and at least one connector:
+* Go to your `site > site settings > Jahia Authentication`
 * In the panel you will see the list of connectors that are available for your site and if you open the card you will see the parameters to fill in order to activate and use it
 * You will need to go to the open authentication website and setup an app to get the parameters. Please refer to the documentation on their website
 * Once this is done a new button will appear `Actions` and if you click on it you will access to the action modules part
```

Comment:
First you enter the parameters (for saml or google-oauth) - when successfully saved, the button "Mappers" appears (not "Actions").
So:
* Once the parameters are successfully saved and the connection is setup, the button 'Mappers' will appear.
* Map at least the mandatory fields from the provider to successfully setup the mapping.